data/json/items/melee/bludgeons.json

```diff
@@ -242,6 +242,25 @@
     "to_hit": 1,
     "category": "weapons"
   },
+  {
+    "id:": "plunger_weapon",
+    "name": { "str": "brain cleaner" },
+    "type": "TOOL",
+    "description": "A barbaric war pick made by driving a metal spike into a toilet plunger. World War IV will be fought with sticks and stones...",
+    "symbol": "/",
+    "color": "red",
+    "weight": "1138",
+    "volume": "2500 ml",
+    "price": "0",
+    "//": "This isn't making a weapon so much as it is ruining a plumbing tool. The sale price has gone down.",
+    "price_postapoc": "90",
+    "to_hit": 0,
```

Comment:
Base: -2
Grip: Solid +1
Length: Short +1
Surface: point -2
Balance: neutral 0
Total is -2, so `"to_hit": -2,`

Reply:
done it